notebooks/243-named-entity-recognition/243-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>For dynamic models it is better to compare performance on sentences of different length.</p><p>I suggest using median inference time:</p><pre class="ql-syntax" spellcheck="false">import numpy as np

inference_times = []
for data in calibration_dataset:
&nbsp; &nbsp; text = ' '.join(data['tokens'])
&nbsp; &nbsp; start = time.perf_counter()
&nbsp; &nbsp; res = ner_pipeline_optimized(text)
&nbsp; &nbsp; end = time.perf_counter()
&nbsp; &nbsp; inference_times.append(end - start)

time_ir = np.median(inference_times)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>